apps/epic-web/src/pages/api/auth/[...nextauth].ts

```diff
@@ -1,11 +1,28 @@
-import NextAuth from 'next-auth'
-import {defaultNextAuthOptions} from '@skillrecordings/skill-api'
+import NextAuth, {NextAuthOptions, Theme} from 'next-auth'
+import {createOptions} from '@skillrecordings/skill-api'
+import {NextApiRequest, NextApiResponse} from 'next'
+import {withSentry} from '@sentry/nextjs'
 
-export const nextAuthOptions = defaultNextAuthOptions({
-  theme: {
-    colorScheme: 'auto',
-    brandColor: '#F59E0B',
-  },
+const productTheme: Theme = {
+  colorScheme: 'auto',
+  brandColor: '#F59E0B',
+}
+
+export const nextAuthOptions: NextAuthOptions = createOptions({
```

Comment:
`createOptions` optionally takes a request so that it can inspect the current url